easypy/resilience.py

```diff
@@ -174,14 +174,15 @@ def retrying(times, acceptable=Exception, sleep=1, max_sleep=False, log_level=lo
     ...         raise Exception('No luck')
     ...     return 'Got lucky'
     """
+
     def wrapper(func):
         @wraps(func)
         def impl(*args, **kwargs):
-            return retry(
-                times, func, args, kwargs,
-                sleep=sleep, max_sleep=max_sleep,
-                acceptable=acceptable, log_level=log_level,
-                pred=pred)
+            retry_params = dict(sleep=sleep, max_sleep=max_sleep, acceptable=acceptable, log_level=log_level, pred=pred)
+            if unacceptable is not False:
```

Comment:
As I explained to @idanarye , In case there will be a change in `retry` default value, it should be transparent to the wrapper, so if no value is passed to it, just trust `retry` to use its own default value instead of set two different by same values.

Reply:
1. When we do these things, we usually use `AUTO`. `None` is also sort-of acceptable because it's used for this purpose all across the Python ecosystem, but `AUTO` is preferable. `False` is not a good choice because `False` does not mean "_don't set it_" - it just means "**don't**".
2. The way to do it is not to add parameter passing logic in every wrapper, but to pass `AUTO` all the way and at the "final" function change it to the actual value:
    ```python
    def retrying(..., unacceptable=AUTO):
        ...
        return retry(..., unacceptable=unacceptable)


    def retry(..., unacceptable=AUTO):
        if unacceptable is AUTO:
            unacceptable = ()
        ...
    ```
3. We usually use this technique when the default value is some sort of a magic number - for example, a timeout, that it is actually feasible that we will want to increase it from 15 to 20 and we don't want to go through many functions all over the codebase and modify them. In this case, the value is not something that's going to change, and even if it will change - we should be more worried about all the places that use it that could be broken due to the change than about this one function right next to it.
4. All the other arguments of `retrying` have their default value right in the function definition without any DRY tricks. Including `sleep` - which is much more susceptible to changes.